cmd/mqtt.go

```
@@ -33,8 +33,8 @@ import (
 )
 
 var (
-	mqttclientid, mqtttopic, defaulttopic, mqttgreylist, gcfgfile, mqttfid string
-	mqttpub, mqttsub, mqttretain, mqttconfigclear                          bool
+	mqttclientid, mqtttopic, defaulttopic, mqttdoubtlist, gcfgfile, mqttfid string
```

golangci-lint (1.62.2)

var `defaulttopic` is unused

(unused)